docs/controllers/scaling.md

```diff
@@ -0,0 +1,115 @@
+# Scaling
+
+This chapter is about strategies for scaling controllers for its final lifecycle stage; where the workload is high and the latencies are important.
```

Comment:
Some wordsmithing suggestions. "Workload" might also be an overloaded term. I tend to think of a workload as something I deploy into the cluster as opposed to the amount of work that a specific application has to deal with, but that's all mainly a me problem :D 

```suggestion
This chapter is about strategies for scaling controllers for their final lifecycle stage; where the workload is high and the latencies are important.
```